.github/workflows/build.yaml

```
@@ -11,17 +11,21 @@ jobs:
       - name: Checkout Packer project
         uses: actions/checkout@v2
 
-      - name: Packer Init
+      - name: Export Application Credentials
         run: |
-          curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
-          sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
-          sudo apt-get update && sudo apt-get install packer
-          packer init .
-
-      - name: Build Packer Artifact
+          echo ${{ secrets.PACKER_SA_KEY }} > $GOOGLE_APPLICATION_CREDENTIALS
```

@obasekiosa check my previous comment for this line. i've even proposed how the code should look like. let me do it again. instead of the `run` section looking like this:
```suggestion
          run: |
             echo ${{ secrets.PACKER_SA_KEY }} > $GOOGLE_APPLICATION_CREDENTIALS
```
you can make it to look like this:
```suggestion
          run: echo ${{ secrets.PACKER_SA_KEY }} > $GOOGLE_APPLICATION_CREDENTIALS
```

---

Oh, I missed that.

I'll add the changes now